src/TargetingPack/Microsoft.Internal.Extensions.Refs/ref/Microsoft.Internal.Extensions.Refs.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
 
     <!-- Do not pack in servicing builds. -->
     <IsPackable>false</IsPackable>
-    <IsPackable Condition=" '$(PatchVersion)' == '0' ">true</IsPackable>
+    <IsPackable Condition=" '$(PatchVersion)' == '1' ">true</IsPackable>
```

Comment:
Change this condition to `$(VersionPrefix)' == '3.0.1'` to guard against accidentally merging this change forwards to 3.1/master